.github/ci/canopennode-rtt/build-stm32f4.sh

```
@@ -645,11 +658,32 @@ verify_profile_object()
+verify_stm32_library_object()
+{
+    local object_name="$1"
+    local object_path
+    local shared_libraries="$RTTHREAD_DIR/bsp/stm32/libraries"
+
+    object_path="$(find "$shared_libraries" -type f -name "$object_name" -print -quit)"
+    if [ -z "$object_path" ]; then
+        echo "Expected STM32 library object was not built: $object_name" >&2
+        exit 1
+    fi
+    log "Profile object: ${object_path#$RTTHREAD_DIR/}"
```

Shellcheck (0.11.0)

[info] Expansions inside ${..} need to be quoted separately, otherwise they match as patterns.

(SC2295)